README.md

```diff
@@ -74,6 +74,8 @@ data_width = 32
 mux_ratio = 2
 write_size = 32
 control = "ReplicaV1"
+# The `pex_level` flag is only available with a full installation.
+pex_level = "rcc"
```

Comment:
here could we briefly list out the constraints on these?
like pex_level = {c, rc, rcc}, mux_ratio = {2,4,8}, num_words = power of 2, etc

Reply:
Updated the README.